modules/vaos/app/services/vaos/appointment_requests_service.rb

```diff
@@ -42,19 +42,16 @@ def deserialize(json_hash)
       []
     end
 
-    def get_appointment_requests_url(start_date = nil, end_date = nil)
-      url = '/var/VeteranAppointmentRequestService/v4/rest/appointment-service'
-      url += "/patient/ICN/#{user.icn}/appointments"
-      url += get_date_url(start_date, end_date).empty? ? '' : "?#{get_date_url(start_date, end_date).to_query}"
-      url
+    def get_appointment_requests_url
+      "/var/VeteranAppointmentRequestService/v4/rest/appointment-service/patient/ICN/#{user.icn}/appointments"
     end
 
-    def get_date_url(start_date, end_date)
-      { startDate: date_format(start_date), endDate: date_format(end_date) }
+    def date_params(start_date, end_date)
+      { startDate: date_format(start_date), endDate: date_format(end_date) }.compact
     end
```

Comment:
This kinda feels like the right time for making a class instead (but I might be missing some context).